02week/pigLatin.js

```diff
@@ -9,15 +9,62 @@ const rl = readline.createInterface({
 
 
 function pigLatin(word) {
+  let lower = word.toLowerCase();
+  let fixed = lower.trim();
+  let space = fixed.indexOf(' ');
+  
+  // let vowel = Math.min(a, e, i, o, u);
+  
+  if (space != -1){
+    let word1 = fixed.substring(0, space);
+    let word2 = fixed.substring(space+1);
+    let latin1 = toLatin(word1);
+    let latin2 = pigLatin(word2);
+    return latin1 + ' ' + latin2;
+  } else {
+    let latin = toLatin(fixed);
+    return latin;
+  }
+  
+}
+
+function toLatin(word){
+  let a = word.indexOf('a');
+  let e = word.indexOf('e');
+  let i = word.indexOf('i');
+  let o = word.indexOf('o');
+  let u = word.indexOf('u');
+  let firstCon = false;
 
-  // Your code here
+  if (a === 0 || e === 0 || i === 0 || o === 0 || u === 0){
+    let latin = word + 'yay';
+    return latin;
+  } else {
+    firstCon = true;
+  }
 
+  if (firstCon === true) {
```

Comment:
you dont really need this check, because if firstCon is false, then the function would have returned above.